### PR TITLE
addition to the junit test

### PR DESCRIPTION
**Pull request by lynnjo**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Added additional asserts to initdb test case
```

### CODE REVIEW

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class InitdbTest {
         }
 
         // Comment these out if you need to look at the logs files created by ProcessBuilder
```
Comment (tcasstevens) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:19]: Comment out what exactly?

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt [hunk 2]
```diff
@@ -38,19 +38,27 @@ class InitdbTest {
         val result = initdb.createDataSets( dbPath)
         println("result = $result")
         // To verify, we  check that the outputDir contains the expected files
-        val expectedFiles = listOf("gvcf_dataset", "hvcf_dataset")
-        expectedFiles.forEach { assert(File(dbPath + it).exists()) }
+        val expectedFiles = listOf("${dbPath}gvcf_dataset", "${dbPath}hvcf_dataset", "${dbPath}temp/tiledb_gvcf_createURI_error.log", "${dbPath}temp/tiledb_gvcf_createURI_output.log", "${dbPath}temp/tiledb_hvcf_createURI_error.log", "${dbPath}temp/tiledb_hvcf_createURI_output.log")
```
Comment (tcasstevens) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:41]: Does dbPath have a / on the end?

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt [hunk 3]
```diff
@@ -38,19 +38,27 @@ class InitdbTest {
...
+        expectedFiles.forEach { assert(File( it).exists()) }
+
+        // Delete the log files created by the ProcessBuilder commands.
+        // Rerun the command .  The log files should NOT be created as
+        // the code will realize the dataset already exists and does not try to recreate it.
+        val expectedLogFiles = listOf("$dbPath}temp/tiledb_gvcf_createURI_output.log", "$dbPath}temp/tiledb_hvcf_createURI_error.log")
```
Comment (tcasstevens) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:47]: Does dbPath have a / on the end?

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt (hunk 1 above)
Comment (lynnjo) [reply src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:19]: Comment out the teardown code.  It removes the folders where the files were written.

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt (hunk 3 above)
Comment (lynnjo) [reply src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:47]: yes - see line 36

**Review by zrm22 — APPROVED** [review]

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt (hunk 3 above)
Comment (pjbradbury) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:47]: Looks like you have unmatched "}"'s here. That is probably not what you want.

**Review by pjbradbury — APPROVED** [review]

File: src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt (hunk 3 above)
Comment (lynnjo) [reply src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt:47]: @pjbradbury Thank you for catching this!  I fixed the test case then neglected to run it once more.  I have made the change and tested.

**Pull request merged**